src/images/icons/planet.svg

```diff
@@ -0,0 +1,5 @@
+<svg xmlns="http://www.w3.org/2000/svg" height="32" width="32" preserveAspectRatio="xMidYMid"
+     xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 32 32">
```

Comment:
we dont' need some of these attributes, please copy from other icons

Reply:
it's from pencil